### ISSUE TITLE
[Bug] profile for University of Washington goes to the home page

### ISSUE THREAD

**Issue by vpchung** [Sage-Bionetworks/sage-monorepo#1466]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What projects are you seeing the problem on?

OpenChallenges

### Current behavior

Clicking on the organization card for UW (University of Washington) goes to the home page.  Looking at the URL, there seems to be a brief pause at `org/uw` but then it redirects to `home`:

https://user-images.githubusercontent.com/9377970/232859736-9b556161-64f0-486d-a5a4-9864891970df.mov

### Expected behavior

Clicking on the organization card for UW should go to the organization profile for UW.

### Anything else?

I'm not sure if this is the case, but perhaps there are some routing conflicts due to another very similar slug? That is, University of Wisconsin-Madison, whose slug is `uw-madison`?

### Are you developing inside the dev container?

- [X] I am using the dev container

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by tschaffter** [comment]
One issue is that the login of organizations is currently limited to 3-25 characters. The login of `uw` in the data is 2 character and so the org service returns a Bad Request error. Same for login that are longer than 25 characters like `arthritis-internet-registry`.

```yaml
description: The login of an organization
type: string
minLength: 3
maxLength: 25
pattern: '^[a-z0-9]+(?:-[a-z0-9]+)*$'
example: example-org
```

A solution is to allow org login between 2 and a larger upper limit, e.g. 64. The longest org login we have on file is `thomas-jefferson-university-hospital` (36 chars).

At the same time I'll add a constraint to the SQL tables.

**Comment by vpchung** [comment]
Thanks, @tschaffter !  Allowing for a minimum of 2 characters would be great -- we have a number of organizations whose acronyms are only 2 letters.

**Comment by tschaffter** [comment]
Great! The fix is implemented by the PR #1482.

**Comment by tschaffter** [comment]
Added to Sprint 23.05
